Automattic/jetpack
# Protect: Add `Summary` component

Add new `Summary` component and introduce it at `protect/admin-page`.

⚠️ *Data is hard-coded. It'll fetch at follow-up PRs* ⚠️ 

#### Changes proposed in this Pull Request:

* Create new `Summary` component.
* Add `protect` at `Storybook` projects path.
* Use `Summary` at `AdminPage`.
* Remove `base-styles` and use `ThemeProvider`.

#### Jetpack product discussion


1202064326332560-as-1202064326332567/f

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `Protect` plugin and activate it.
* Navigate to `protect` homepage.
* You should see summary.

#### Demo

![Screen Shot 2022-04-04 at 20 31 02](https://user-images.githubusercontent.com/1663717/161649824-448d1e1d-7cb3-4877-a875-d8cffd1cada8.png)
